packages/mdc-menu/foundation.js

```diff
@@ -208,6 +210,11 @@ class MDCMenuFoundation extends MDCFoundation {
     this.setSelectedIndex(-1);
   }
 
+  /** @param {boolean} quickOpen */
+  quickOpen(quickOpen) {
```

Comment:
Should this be called `setQuickOpen`? The current name in the context of a method makes it sound like it's actually going to open the menu.